native/cocos/renderer/pipeline/custom/FrameGraphDispatcher.cpp

```diff
@@ -90,6 +90,23 @@ void FrameGraphDispatcher::setParalellWeight(float paralellExecWeight) {
     _paralellExecWeight = clampf(paralellExecWeight, 0.0F, 1.0F);
 }
 
+const ResourceAccessNode &FrameGraphDispatcher::getAttachmentStatus(RenderGraph::vertex_descriptor renderGraphVertID) const {
+    auto iter = resourceAccessGraph.subpassIndex.find(renderGraphVertID);
+    auto ragVertID = resourceAccessGraph.passIndex.at(renderGraphVertID);
+    const ResourceAccessNode* accessNode = &resourceAccessGraph.access.at(ragVertID);
```

Comment:
Can get `accessNode` in `if` sentence as it is only used there.

Reply:
Here `accessNode` is initialized with a default value. It might be updated in the `if` statement.
So it cannot be moved into the `if` statement.

Reply:
It is my fault. It will return the value, not just be used in `if`.